index.bs

```diff
@@ -783,27 +803,40 @@ the maximum number of attributions for a ([=attribution rate-limit record/source
 [=attribution rate-limit record/attribution destination=],
 [=attribution rate-limit record/reporting endpoint=]) per [=attribution rate-limit window=].
 
-<h3 dfn id="should-block-attribution-for-reporting-endpoint-limit">Should attribution be blocked by reporting-endpoint limit</h3>
+<h3 dfn id="should-block-processing-for-reporting-endpoint-limit">Should processing be blocked by reporting-endpoint limit</h3>
 
-Given an [=attribution trigger=] |trigger| and [=attribution source=] |sourceToAttribute|:
+Given an [=attribution rate-limit record=] |newRecord|:
 
-1. Let |matchingRateLimitRecords| be all entries in the [=attribution rate-limit cache=] where all of the following are true:
-     * entry's [=attribution rate-limit record/source site=] and |sourceToAttribute|'s [=attribution source/source site=] are equal
-     * entry's [=attribution rate-limit record/attribution destination=] and |trigger|'s [=attribution trigger/attribution destination=] are equal
-     * entry's [=attribution rate-limit record/trigger time=] is at least [=attribution rate-limit window=] before |trigger|'s [=attribution trigger/trigger time=]
+1. Let |max| be [=max source reporting endpoints per rate-limit window=].
```

Comment:
Another option here would be to put the limits in a map and do a lookup, but I think this is just as clear.